### ISSUE TITLE
Warning on the YAML flags with `type: boolean`

### ISSUE THREAD

**Issue by merybenavente** [guildai/guildai#56]
Example of an operation:
```
operation:
  train:
    flags:
      generate_pos:
          type: boolean
          default: false
          choices: [true, false]
```
Warning output: `WARNING: unknown flag type 'boolean' for generate_pos - cannot coerce`.

**Comment by gar1t** [comment]
Thank you! This will be resolved in 0.6.7 - an early release version is available now.

**Comment by gar1t** [comment]
I created an issue resolution project for this:

https://github.com/guildai/issue-resolution/tree/master/56-warning-flags-with-type-boolean

The fix is available in the current pre-release and will officially land in 0.7.0.

**Comment by gar1t** [comment]
The fix is now generally available in 0.7.0. If this is still an issue, please re-open and we'll sort it out. Thanks again for the report.

**Comment by merybenavente** [comment]
thank you!
